GDJS/Runtime/variable.js

```diff
@@ -92,11 +92,11 @@ gdjs.Variable.prototype.getChild = function(childName) {
 };
 
 /**
- * Get the child with the specified name.
+ * Return the child in a variable.
  *
- * If the variable has not the specified child, an empty variable with the specified name
- * is added as child.
- * @method getChild
+ * If the variable has the specified children
```

Comment:
Only the last sentence should be preserved:
`Check if the variable has the specified children`.